### ISSUE TITLE
[Bug]: AE2 Meteorite Sites Not Generating In Single Player

### ISSUE THREAD

**Issue by KirinDave** [FTBTeam/FTB-Modpack-Issues#854]
### Modpack

Direwolf20 1.18

### Modpack version

1.2.0

### Describe the bug

After testing several single player worlds, it appears that AE2 meteors are not spawning. The meteorite compass simply spins and I can find no evidence the generation has ever happened.

### Steps to reproduce

1. Create A Single Player World (w/ cheats enabled)
2. Spawn in a meteorite compass
3. Receive no signal
4. Teleport 5000 blocks in a random direction
5. Goto 3

### Expected behaviour

Usual mod structure sites would be generated

**Comment by quietust** [comment]
In this version, the meteorite compass only seems to point at a target once the chunks have actually been generated - it's not like throwing Eyes of Ender or using Nature's Compass where it can find things anywhere in the world.

**Comment by KirinDave** [comment]
I've got some free time, I'll try and verify this. I did actually cover a fair amount of ground without simply teleporting.

**Comment by KirinDave** [comment]
I have a seed I believe is an example of the problem: -1007143982

Further updates though, I am suspicious that this is a 1.0.0->1.2.0 issue. New maps I make fresh with a fresh 1.2.0 instance _seem_ to have meteors. Remaking maps generated with 1.0.0 (i.e., the seed above) do not appear to have meteors (and may be missing other structures as well? I notice a lack of mineshafts I've seen in YT playthroughs). So either I've been very specifically lucky in my tests so far or something else is going on.

**Comment by KirinDave** [comment]
With help from the discord I think I've worked this out. Part of the problem is the compass doesn't locate worldgen, it locates skystone blocks, so we can't 

Some seeds (e.g., the one I gave) generated in 1.0) do not allow meteorites to spawn near player spawn. Using `/locate` reveals locations where spawning was expected but fails. I'm not sure why and I suspect this is a mod interaction bug. Another popular pack has also run into this with 1.18 and they're not really sure how it happens either.

The fix is to move *very* far away from spawn (it only started working for me with values over 3000. to get to fresh chunks then use `/locate ae2:meteorite`. This should help direct you to a meteorite that generates in 1.2 or 1.3 far away, which probably won't be visible via the compass initially.

I will leave it up to the staff if they want to close this bug or take up the worldgen collision issues with the mod authors. I consider my issue solved and thansk for quietust for giving me the tip about the compass.
